projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -804,12 +804,12 @@ export class IgxOverlayService implements OnDestroy {
                 return;
             }
             if (info.settings.closeOnOutsideClick) {
-                const target = ev.target as any;
+                const target = ev.target as Node;
                 const overlayElement = info.elementRef.nativeElement;
                 // check if the click is on the overlay element or on an element from the exclusion list, and if so do not close the overlay
-                const excludeElements = info.settings.excludeFromOutsideClick ?
+                const excludeElements: HTMLElement[] = info.settings.excludeFromOutsideClick ?
```

Comment:
`excludeFromOutsideClick` is already typed as `HTMLElement[]` so this shouldn't be needed. In fact I think with this version the issue is no longer reproducible anyway